charts/performancetesting/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "chartest.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
```

Comment:
Is this `_helpers.tpl` file being used elsewhere? Upon first glance it looks like it's not required and may be able to be removed.

Am I correct that it defines custom values that are prefixed with `chartest.{variable_name}`? If so, doesn't look like `chartest.` is being referenced anywhere, and this file could be deleted.

Reply:
You  rigth, I´ll remove it.